webrender/src/prim_store.rs

```diff
@@ -1647,6 +1656,19 @@ impl PrimitiveStore {
                     (Some((pic.pipeline_id, mem::replace(&mut pic.runs, Vec::new()), rfid)),
                      pic.cull_children,
                      may_need_clip_mask)
+                },
+                PrimitiveKind::Brush => {
+                    let brush = &mut self.cpu_brushes[metadata.cpu_prim_index.0];
+                    if let BrushKind::Solid { ref mut color, ref binding } = brush.kind {
+                        *color = scene_properties.resolve_color(&binding);
```

Comment:
Maybe I should add an `if let &PropertyBinding::Binding(_) = binding` here so I'm not trying to resolve `PropertyBinding::Value` properties?